test/bookstore/gce/e2e.sh

```diff
@@ -82,13 +82,23 @@ run cat "${VM_STARTUP_SCRIPT}"
 trap cleanup EXIT
 
 # Creating the VM
-run retry -n 3 gcloud compute instances create "${INSTANCE_NAME}" \
-  --machine-type "custom-2-3840" \
-  --image-family "${VM_IMAGE}" \
-  --image-project debian-cloud \
-  --boot-disk-size "100GB" \
-  --metadata endpoints-service-name="${ESP_SERVICE}",endpoints-service-version="${ESP_SERVICE_VERSION}" \
-  --metadata-from-file startup-script="${VM_STARTUP_SCRIPT}"
+if [[ "${ROLLOUT_STRATEGY}" == "original" ]]; then
+  run retry -n 3 gcloud compute instances create "${INSTANCE_NAME}" \
+    --machine-type "custom-2-3840" \
+    --image-family "${VM_IMAGE}" \
+    --image-project debian-cloud \
+    --boot-disk-size "100GB" \
+    --metadata endpoints-service-name="${ESP_SERVICE}",endpoints-service-version="${ESP_SERVICE_VERSION}" \
+    --metadata-from-file startup-script="${VM_STARTUP_SCRIPT}"
+else
+  run retry -n 3 gcloud compute instances create "${INSTANCE_NAME}" \
+    --machine-type "custom-2-3840" \
+    --image-family "${VM_IMAGE}" \
+    --image-project debian-cloud \
+    --boot-disk-size "100GB" \
+    --metadata endpoints-service-name="${ESP_SERVICE}",endpoints-rollout-strategy="${ROLLOUT_STRATEGY}" \
```

Comment:
can we add:  push a new config, and make sure new config is not applied or applied based on the strategy.